kclvm/sema/src/namer/mod.rs

```diff
@@ -0,0 +1,149 @@
+use crate::core::global_state::GlobalState;
+use crate::core::package::PackageInfo;
+use crate::core::symbol::{PackageSymbol, SymbolRef};
+use kclvm_ast::ast::Program;
+use kclvm_ast::walker::MutSelfTypedResultWalker;
+use kclvm_error::Position;
+mod node;
+
+struct Namer<'ctx> {
```

Comment:
Would U pls can add more documents or link the design doc into the issue https://github.com/kcl-lang/kcl/issues/742 ?

Besides, `Namer` is a private structure but its method `find_symbols` is a public function.